rbs/rbs_common.cc

```diff
@@ -34,4 +34,21 @@ core::LocOffsets RBSDeclaration::typeLocFromRange(const rbs_range_t &range) cons
     }
     return comments.front().typeLoc;
 }
+
+bool isVisibilitySend(const parser::Send *send) {
```

Comment:
Now used by both Associator and AssertionsRewriter

Reply:
Let's move it in https://github.com/sorbet/sorbet/blob/master/parser/helper.h#L365 instead

Reply:
Yes I think it needs to be static